extensions/src/AWSSDK.Extensions.NETCore.Setup/ClientFactory.cs

```diff
@@ -79,18 +79,18 @@ internal object CreateServiceClient(IServiceProvider provider)
         /// </summary>
         /// <param name="provider">The dependency injection provider.</param>
         /// <returns>The AWS service client</returns>
-        internal static IAmazonService CreateServiceClient(ILogger logger, Type serviceInterfaceType, AWSOptions options)
+        internal IAmazonService CreateServiceClient(ILogger logger, AWSOptions options)
```

Comment:
`<param name="provider">` wasn't updated to reflect the parameters change

Reply:
Fixed